### DISCUSSION TITLE
Language of text files in slang

### DISCUSSION THREAD

**Discussion by riscait** [altive/flutter_app_template#288] [Polls]
The default is JSON, but YAML and CSV are selectable.

**Comment by naipaka** [comment]
JSONだと`""`で毎回囲まなくてはいけないのが煩わしそうなのでYAML派です！
あとslangで書くかわからないですが、コメント書けるのも利点ですかね✍️
CSVは階層構造で可読性悪くなりそうかも💭

**Comment by k-nkmr** [comment]
コメント書けるのが便利なのでYAMLがいいかなと思っています📝

**Comment by riscait** [comment]
お二人とも、投票とコメントありがとうございます🙌
YAMLでいきましょう！！
JSONから変更するIssueも作成しました！ https://github.com/altive/flutter_app_template/issues/290
